server/app.js

```diff
@@ -1,3 +1,4 @@
+import 'dotenv/config';
```

Comment:
With the new default env vars, there's nothing we need to add to get email working in development with mailcatcher. However, if we want to use dotenv in the future, it should be run/configured at the earliest possible opportunity, so I've moved it to the beginning of app.js itself.